Dockerfile

```diff
@@ -0,0 +1,32 @@
+
+FROM debian:10 as build
+
+ARG release=0.3.0
+
+USER root
+WORKDIR /tmp
+ENV DEBIAN_FRONTEND=noninteractive
+
+RUN apt-get update -q -y && apt-get install -y -q \
+      unzip wget
+RUN wget -q -O the_archive.zip "https://github.com/glencoesoftware/raw2ometiff/releases/download/v${release}/raw2ometiff-${release}.zip"
```

Comment:
Happy to defer, but my preference here and the style we've followed with the github.com/ome docker images is that if a Dockerfile is in a source repo, then it builds the local code, and if it's pulling an artifact, then it sits in a separate repo.

Reply:
Let me know what you prefer.  If you point me to the build commands for raw2ometiff I can integrate them into the Dockerfile (or you could make changes directly to the PR).  Alternatively, I could re-create this PR against a different repository.

Reply:
`gradle build` as in https://github.com/glencoesoftware/bioformats2raw/blob/master/Dockerfile#L27 should suffice.

Reply:
done!